charter.md

```diff
@@ -53,15 +53,245 @@ The following responsibilities belong to the Steering Committe, but may be deleg
 * Control and delegate access to and establish processes regarding
   project resources/assets<sup>[6](#footnote6)</sup>  
 
-## Quorum
+## Changes
 
-Steering committee members are generally expected to attend every meeting. We
+In instances where a process is not already specified within this document,
+changes to the Steering Committee charter will be considered according to the
+processes set forth in the committee's [operations documentation][changes].
+
+[changes]: /operations/changes.md
+
+## Membership
+
+### Composition
+
+The Steering Committee is composed of seven (7) members.
+
+### Elections
+
+<!--
+
+- Consider explicitly mentioning SIG ContribEx's Elections subproject, once it
+exists.
+
+-->
+
+Every year, the Steering Committee holds a general election for open seats with
+[SIG Contributor Experience][sig-contribex] owning the process.
+
+Our [election policy document][general-elections] covers the details for how
+this works.
+
+In the instance that a general election is called as a result of the
+[dissolution](#dissolution) of the current committee, the following rules will
+apply:
+
+* members of the current committee will not be allowed to run in the new
+  election
```

Comment:
Perhaps noting that there *is* a plan for continuity below would be helpful; personally, I'd like to know that there's still a plan to provide support to new members (considering that existing steering members are unable to run).

I think the thing that isn't clear for me here, though, is whether those members are unable to run once or forever. Is the intention to say the steering members who were on the disbanded committee can never run again, or only for this election and then they're eligible for the next election?

Reply:
I believe the intention is once, and I agree it'd be good to state that explicitly.